utils/training_helper.py

```diff
@@ -83,7 +86,7 @@ def __init__(self, units, binf_to_ipa=None, inner_projection_layer=True, **kwarg
 
     def call(self, inputs):
         if self.inner_projection_layer:
-            outputs = super().call(inputs)
+            outputs = super(DenseBinfDecoder, self).call(inputs)
```

Comment:
Do we support Python 2? :)

Reply:
Nope, yet this line crashes in my local Python 3.6 with runtime error if no super params are given.